docs/guide/object-floors.md

```diff
@@ -157,6 +157,8 @@ UnevenTerrain {
   SFBool     flatCenter     FALSE
   SFBool     flatBounds     FALSE
   SFInt32    perlinNOctaves 3
+  deprecatedField MFString   texture        "textures/sand.jpg"
+  deprecatedField SFVec2f    textureScale   1.0 1.0
```

Comment:
I am not sure we should document the deprecated fields...
```suggestion
```

Reply:
Good point, I will improve the script to avoid this.

Reply:
This is now done in cc4cfddbd34415909867a40d3222fae702fdb888.